### DISCUSSION TITLE
cms integration

### DISCUSSION THREAD

**Discussion by wanghaisheng** [arthelokyo/astrowind#168] [Q&A]
in this doc ,https://docs.astro.build/zh-cn/guides/cms/decap-cms/

>Use the [astro-netlify-cms](https://github.com/delucis/astro-netlify-cms) Astro integration to add Netlify CMS to your project.

and here https://decapcms.org/docs/github-backend/
>For repositories stored on GitHub, the github backend allows CMS users to log in directly with their GitHub account. Note that all users must have push access to your content repository for this to work.

does this mean I create a new repo for blog content management  or  create a  folder  like public under astrowind source code and manage them through this  cms ui

**Comment by widgeter** [comment]
Hi @wanghaisheng 

We have added a minimal example configuration for DecapCMS related to posts:
https://github.com/onwidget/astrowind/commit/0b89a3b22b42173f25885ca2194094823614b461

Also with some small changes it is possible to have access to the edition of the content of the pages (not included at this moment). For this, all the content of the pages should be in the `src/content` folder and referenced from each page.
We are evaluating the easiest way to do this for future versions of **AstroWind**.

Any questions do not hesitate to contact us.

**Reply by wanghaisheng** [reply]
wow  thanks for your information
